docs/getting_started/client.md

```diff
@@ -28,11 +28,20 @@ By default, the client will query all OPTIMADE API URLs that it can find via the
     results = client.get()
     ```
 
+At the command line, it may be immediately useful to redirect or save these results to a file (see [Accessing the results](#accessing-the-results):
```

Comment:
I think you can leave out "immediately" I do not see what it adds to the sentence